modules/37_industry/subsectors/datainput.gms

```diff
@@ -49,9 +57,144 @@ if (cm_IndCCSscen eq 1,
+p37_energy_limit("ue_cement")          =  10000;
+p37_energy_limit("ue_steel_primary")   =  10000;
+p37_energy_limit("ue_steel_secondary") = 100000;
+
+*' Emission factors for calculating industry emissions
+p37_fctEmi("fesos") = fm_dataemiglob("pecoal","sesofos", "coaltr","co2");
+p37_fctEmi("fehos") = fm_dataemiglob("peoil", "seliqfos","refliq","co2");
+p37_fctEmi("fegas") = fm_dataemiglob("pegas", "segafos", "gastr", "co2");
+
+*** CCS for industry is off by default
+emiMacSector(emiInd37_fuel) = NO;
+pm_macSwitch(emiInd37)      = NO;
+
+*** turn on CCS for industry emissions
+if (cm_IndCCSscen eq 1,
+  if (cm_CCS_cement eq 1,
+    emiMacSector("co2cement") = YES;
+    pm_macSwitch("co2cement") = YES;
+    pm_macSwitch("co2cement_process") = YES;
+    emiMac2mac("co2cement","co2cement") = YES;
+    emiMac2mac("co2cement_process","co2cement") = YES;
+  );
+
+  if (cm_CCS_chemicals eq 1,
+    emiMacSector("co2chemicals") = YES;
+    pm_macSwitch("co2chemicals") = YES;
+    emiMac2mac("co2chemicals","co2chemicals") = YES;
+  );
+
+  if (cm_CCS_steel eq 1,
+    emiMacSector("co2steel") = YES;
+    pm_macSwitch("co2steel") = YES;
+    emiMac2mac("co2steel","co2steel") = YES;
+  );
+);
+
+*** CCS for other industry is off in any case
+emiMacSector("co2otherInd") = NO;
+pm_macSwitch("co2otherInd") = NO;
+emiMac2mac("co2otherInd","co2otherInd") = NO;
+
+*** data on maximum secondary steel production
+Parameter 
+  p37_cesIO_up_steel_secondary(tall,all_regi,all_GDPscen)   "upper limit to secondary steel production based on scrap availability"
+  /
+$ondelim
+$include "./modules/37_industry/subsectors/input/p37_cesIO_up_steel_secondary.inc";
+$offdelim
+  /
+;
+
+s37_clinker_process_CO2 = 0.5262;
+
+*** FIXME *** this needs to be in moinput
+p37_clinker_cement_ratio("2005","CAZ") = 0.81;
+p37_clinker_cement_ratio("2005","CHA") = 0.58;
+p37_clinker_cement_ratio("2005","EUR") = 0.73;
+p37_clinker_cement_ratio("2005","IND") = 0.71;
+p37_clinker_cement_ratio("2005","JPN") = 0.80;
+p37_clinker_cement_ratio("2005","LAM") = 0.70;
+p37_clinker_cement_ratio("2005","MEA") = 0.81;
+p37_clinker_cement_ratio("2005","NEU") = 0.81;
+p37_clinker_cement_ratio("2005","OAS") = 0.80;
+p37_clinker_cement_ratio("2005","REF") = 0.80;
+p37_clinker_cement_ratio("2005","SSA") = 0.77;
+p37_clinker_cement_ratio("2005","USA") = 0.82;
+
+*' Clinker-to-cement ratios converge to the lowest regional 2005 value by 2100.
+p37_clinker_cement_ratio(t,regi)
+  = p37_clinker_cement_ratio("2005",regi)
+  + ( smin(regi2, p37_clinker_cement_ratio("2005",regi2))
+    - p37_clinker_cement_ratio("2005",regi)
+    )
+  * (min(t.val, 2100) - 2005)
+  / (2100             - 2005);
+
+*** Cement demand reduction is implicit in the production function, so no extra
+*** costs have to be calculated.
+pm_CementDemandReductionCost(ttot,regi) = 0;
+
+if (cm_emiscen ne 1,
+  Execute_Loadpoint "input_ref.gdx" p37_cesIO_base = vm_cesIO.l;
+);
+
+*** FIXME calibration debug
```

Comment:
Is it something you would like to deal with before the merge?

Reply:
Not really.